SwerveControllerCommand/robot.py

```diff
@@ -0,0 +1,66 @@
+"""
+ Copyright (c) FIRST and other WPILib contributors.
+ Open Source Software; you can modify and/or share it under the terms of
+ the WPILib BSD license file in the root directory of this project.
+"""
+
+from wpilib import TimedRobot
+import commands2
+from robotcontainer import RobotContainer
+
+class Robot(TimedRobot):
```

Comment:
RobotPy command robots should inherit from TimedCommandRobot

Reply:
Sorry, about that, the original file inherited from TimedRobot.